app/src/main/java/chat/rocket/android/helper/Avatar.java

```diff
@@ -73,7 +71,7 @@ private String getImageUrl() {
     //from Rocket.Chat:packages/rocketchat-ui/lib/avatar.coffee
     //REMARK! this is often SVG image! (see: Rocket.Chat:server/startup/avatar.coffee)
     try {
-      final String avatarUrl = "/avatar/" + URLEncoder.encode(username, "UTF-8") + ".jpg";
+      final String avatarUrl = "/avatar/" + URLEncoder.encode(username, "UTF-8");
```

Comment:
Adding just for future reference, all data that reaches an `RecyclerView.Adater` should be ready to be presented, doing a lot of short-lived small memory allocations on the adapter adds pressure do the Garbage Collector.